app/styles/upf-variables.less

```diff
@@ -1,7 +1,7 @@
 // Inverse-element
 @inverse-bg: @navbar-default-bg;
 @inverse-btn-active-color: @dark-blue-hover;
-@inverse-color: @white;
+@inverse-color: fade(white, 60%);
```

Comment:
If the `inverse-*` variables are only used in the `inverse` class, why should we bother set them all as variables ?